roles/setup/tasks/python_packages.yml

```diff
@@ -0,0 +1,29 @@
+- name: Get the python version
+  ansible.builtin.command: >
+    {{ python_executable }} -c 'import sys; print(sys.version.split()[0])'
+  register: python_version
+  changed_when: false
+
+- name: Confirm python version is acceptable
+  ansible.builtin.fail:
+    msg: "Python version {{ minimum_python_version }} or greater is required, you have {{ python_version.stdout }}"
+  when: python_version.stdout is version(minimum_python_version, '<')
+
+- name: Append receptorctl to python packages
+  ansible.builtin.set_fact:
+    additional_python_packages: "{{ additional_python_packages + ['receptorctl'] }}"
```

Comment:
why are we forcing the installation of receptorctl?

Reply:
the Reload Receptor handler needs it to call `receptorctl reload`